### PR TITLE
plugin SDK link in docs, use SDK in plugin template

### PR DESCRIPTION
**Pull request by keksa**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -6,10 +6,14 @@ Plugins can be used as:
 * frontend pages - a completely new page with own features and UI can be shown to authenticated UCRM users or clients. This can be used for a custom features, for example user-defined reports, batch data changes with user input, etc.
 
 ## Plugin samples for an easy start of creating your own plugin
-* [Plugin Template](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/examples/plugin-template) - Simpliest UCRM plugin sample, can be used as a base for a new plugin development.
+* [Plugin Template](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/examples/plugin-template) - Simplest UCRM plugin sample, can be used as a base for a new plugin development.
 * [Invoice CSV Export](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/invoice-csv-export) - Better plugin example, actually doing something: a new menu item shown, enables user to define From & To date filters, exports filtered invoices into CSV.
```
Comment (Piskvor) [inline docs/index.md]: *does something

File: docs/index.md [hunk 2]
```diff
@@ -6,10 +6,14 @@ Plugins can be used as:
...
 
 ## Plugin samples for an easy start of creating your own plugin
-* [Plugin Template](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/examples/plugin-template) - Simpliest UCRM plugin sample, can be used as a base for a new plugin development.
+* [Plugin Template](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/examples/plugin-template) - Simplest UCRM plugin sample, can be used as a base for a new plugin development.
 * [Invoice CSV Export](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/invoice-csv-export) - Better plugin example, actually doing something: a new menu item shown, enables user to define From & To date filters, exports filtered invoices into CSV.  
 * [Revenue Report](https://github.com/Ubiquiti-App/UCRM-plugins/tree/master/plugins/revenue-report) - Revenue report grouped by products or services, shown under the Reporting main menu section.
 
+## UCRM Plugin SDK
+You can use [UCRM Plugin SDK](https://github.com/Ubiquiti-App/UCRM-Plugin-SDK) to help you with development of plugins.  
+It contains classes able to help you with calling UCRM API, getting plugin's configuration and much more.
```
Comment (Piskvor) [inline docs/index.md]: Too complicated, refactor:
"It contains classes to call UCRM API, to get plugin's configuration, and more."

File: docs/index.md [hunk 3]
```diff
@@ -6,10 +6,14 @@ Plugins can be used as:
...
+
 ## Distribution
 Plugins are distributed as ZIP archives. User uploads the archive into UCRM, it's validated and if valid, extracted to a folder based on plugin's name.
```
Comment (Piskvor) [inline docs/index.md]: s/validated/checked/ ?

File: examples/plugin-template/src/composer.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "description": "This is plugin template.",
```
Comment (Piskvor) [inline examples/plugin-template/src/composer.json]: "This is a template you can use to build your plugins"?

File: examples/plugin-template/src/composer.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+    "license": "MIT",
+    "require": {
+        "ubnt/ucrm-plugin-sdk": "~0.3"
```
Comment (Piskvor) [inline examples/plugin-template/src/composer.json]: "We expect major changes going to 1.0.0"...okay.

File: pack-plugin.php [hunk 6]
```diff
@@ -30,7 +40,7 @@
 
 if ($zip->open($zipFile, ZipArchive::CREATE) !== true) {
     echo 'Can\'t open zip file.' . PHP_EOL;
-    exit(3);
+    exit(4);
```
Comment (Piskvor) [inline pack-plugin.php]: Have we released this yet? Compat break!

File: pack-plugin.php [hunk 7]
```diff
@@ -59,7 +69,7 @@ function (SplFileInfo $fileInfo) {
 
     if (! $zip->addFile($fileInfo->getPathname(), ltrim($filename, '/'))) {
         echo sprintf('Unable to add file "%s".', $filename) . PHP_EOL;
-        exit(4);
+        exit(5);
```
Comment (Piskvor) [inline pack-plugin.php]: Dtto. (Wufo this change, anyway?)

File: pack-plugin.php (hunk 6 above)
Comment (keksa) [reply pack-plugin.php]: released today, actually changed all the codes to 1 instead

**Pull request merged**